packages/auto-chart/src/AdviceList/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { prefixCls, getThumbnailURL, customChartType } from '../utils';
+
+interface AdviceListProps {
+  advices: any[];
+  currentIndex: number;
+  containerHover: boolean;
+  onChartChange: (valueIndex: number) => void;
```

Comment:
```suggestion
  onChartTypeChange: (valueIndex: number) => void;
```

这里 chart list 主要是修改图表类型的，如果是 config panel 也会造成 chart change，可以从命名上区分开

Reply:
名称已修改